kotlin/workflow-ui-android/src/test/java/com/squareup/workflow/ui/WorkflowRunnerViewModelTest.kt

```diff
@@ -44,13 +59,40 @@ class WorkflowRunnerViewModelTest {
     scope.coroutineContext[Job]!!.invokeOnCompletion { e ->
       if (e is CancellationException) cancelled = true
     }
-    val runner = WorkflowRunnerViewModel(viewRegistry, emptyFlow(), emptyFlow(), scope)
 
-    assertThat(cancelled).isFalse()
-    runner.output.test()
+    val runner = WorkflowRunnerViewModel(viewRegistry, emptyFlow(), MutableLiveData(), scope)
+
     assertThat(cancelled).isFalse()
 
     runner.clearForTest()
     assertThat(cancelled).isTrue()
   }
+
+  @Test fun magilla() {
+    val outputs = ConflatedBroadcastChannel<String>()
+
+    val w = object : StatelessWorkflow<Unit, String, Unit>() {
+      override fun render(
+        input: Unit,
+        context: RenderContext<Nothing, String>
+      ) {
+        context.onWorkerOutput(outputs.asWorker()) { emitOutput(it) }
+      }
+    }
+
+    val factory = WorkflowRunnerViewModel.Factory(
+        w, viewRegistry, flowOf(Unit), null, TestCoroutineDispatcher()
+    )
+    val model = factory.create(WorkflowRunnerViewModel::class.java)
+
+    runBlockingTest {
```

Comment:
Why are you using `runBlockingTest` instead of just `runBlocking`? The test infrastructure is generally very over-engineered for anything we need it for.

Reply:
Vestige of first attempt to get a `TestCoroutineDispatcher` (or even discover it).

Reply:
Why do you need a test dispatcher? Unless you're doing delays, you probably don't need it.